app/(main)/(screens)/apply-now/preview.tsx

```diff
@@ -46,23 +68,28 @@ const Preview = () => {
         return;
       }
 
-      //fetch current logged in user
-
-      const user = AuthApp.currentUser;
-
       if (!user) {
         setErrorMessage("User not found");
         console.error("User not found");
         return;
       }
 
+      console.log("Submitting application with data:", {
```

Comment:
Let's remove console logs from everywhere, and don't add them in future as well. As it will become hard to debug if we keep consoles everywhere.